Formula/lv2.rb

```diff
@@ -60,15 +60,15 @@ class Lv2 < Formula
 
   def install
     # Python resources and virtualenv are for the lv2specgen.py script that is installed
-    venv = virtualenv_create(libexec, Formula["python@3.9"].opt_bin/"python3")
+    venv = virtualenv_create(libexec, "python3")
     venv.pip_install resources
     rw_info = python_shebang_rewrite_info("#{libexec}/bin/python3")
     rewrite_shebang rw_info, *Dir.glob("lv2specgen/*.py")
 
-    system Formula["python@3.9"].opt_bin/"python3", "./waf", "configure",
-           "--prefix=#{prefix}", "--no-plugins", "--lv2dir=#{lib}"
-    system Formula["python@3.9"].opt_bin/"python3", "./waf", "build"
-    system Formula["python@3.9"].opt_bin/"python3", "./waf", "install"
+    system "python3", "./waf", "configure",
+           "--prefix=#{prefix}", "--no-plugins", "--lv2dir=#{lib}/lv2"
```

Comment:
Won't this still install a user's plugins into the Cellar, since `lib/"lv2"` is located in the Cellar?

Reply:
It will, but this gets symlinked up into `HOMEBREW_PREFIX/lib/lv2` anyways (at least this was the behavior when I tried it locally). Users can drop other plugins (not managed by Homebrew) into `HOMEBREW_PREFIX/lib/lv2` as well. Or am I missing something here?

Reply:
I guess what I'm confused about is that passing `lib/"lv2"` means that `lv2` will know to look in `HOMEBREW_CELLAR/"lv2"/version/"lib/lv2"` but not necessarily in `HOMEBREW_PREFIX/"lib/lv2"`. I feel like I'm missing something here?

Reply:
Ah, ok. `--lv2dir=` controls the directory where the core LV2 plugins/definitions (part of the `lv2` formula) are placed. These plugins/definitions are used by an LV2 _host_ (e.g. Audacity, Reaper, some other DAW or CLI tool). The `lv2` formula doesn't actually use any LV2 plugins; it doesn't need to look for plugins anywhere.

In fact, I think `lv2` might be a build-time (not runtime) dependency for other LV2 plugins like `mda-lv2`. Need more investigation on that front.

LV2 plugins are relocatable (e.g. they could also be placed in `~/.lv2`).